yatai/helm/yatai/values.yaml

```diff
@@ -58,3 +58,8 @@ yatai-image-builder:
     serviceAccountName: "yatai"
   dockerRegistry:
     secure: true
+
```

Comment:
We should add default memory requests and limits and a default CPU requests for the yatai components.